src/Support/SelectFields.php

```diff
@@ -213,7 +213,7 @@ protected static function handleFields(array $queryArgs, array $requestedFields,
                 // Select
                 else {
                     $key = isset($fieldObject->config['alias'])
-                        ? $fieldObject->config['alias']
+                        ? value($fieldObject->config['alias'])
```

Comment:
I know these global helpers are super convenient, but the same reason they're global I despise them.

In this case this should be replaced with inline code doing the same.